### DISCUSSION TITLE
Optoma projector

### DISCUSSION THREAD

**Discussion by GGORG0** [probonopd/irdb#94] [Q&A]
Anyone has codes for the Optoma projectors? I can't find them in the database.

**Comment by bengtmartensson** [comment]
Firing up IrScrutinizer and RemoteLocator therein (in the Import pane) shows that there are a number of JP1 files for Optoma projectors. Download one such (need to register in the JP1 forum for this), open it in current RemoteMaster, export as Girr. This file contains your data. If you need more help, just ask.
